common/djangoapps/third_party_auth/samlproviderconfig/tests/test_samlproviderconfig.py

```diff
@@ -23,7 +23,8 @@
     'metadata_source': 'http://test.url',
     'name': 'name-of-config',
     'enabled': 'true',
-    'slug': 'test-slug'
+    'slug': 'test-slug',
+    'country': 'https://example.customer.com/countrycode'
```

Comment:
Can we try to add a test where the country is 
(1) not provided in the input data when posting ?  that will ensure we get the default expected value for it ( empty string ?) 
(2) provided, but is not a urn (sth like 'abc'),  that will ensure ??? not sure what we should expect here but curious

Ideally that kind of test would be a smaller unit test for the model but I dn't see one setup like that anywhere .. so it may be fine to add such a test here ? thoughts ?

Reply:
Yup! I feel like empty string default is good here, especially since many IDP's don't provide country (at least from what I'm told?). 

Parameter format validation is an interesting subject. I support the idea of validating the urns but slightly unsure of what the proper approach is. 

I'm ok with adding country related smaller tests to the `SAMLProviderConfigTests` class.

Reply:
re @georgebabey offline- `"I don’t think you need to do any fancy validation. We also can’t assume they’re using URNs, some SAML providers just use strings for the key"`

Reply:
this also follows why none of the other URN fields have validation for such. 🤷